modules/secure-cloud-run-core/variables.tf

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "default_rules" {
+  description = "Default rule for Cloud Armor."
+  default = {
+    default_rule = {
+      action         = "allow"
+      priority       = "2147483647"
+      versioned_expr = "SRC_IPS_V1"
+      src_ip_ranges  = ["*"]
+      description    = "Default allow all rule"
+    }
+  }
+  type = map(object({
+    action         = string
+    priority       = string
+    versioned_expr = string
+    src_ip_ranges  = list(string)
+    description    = string
+  }))
+}
+
+variable "owasp_rules" {
+  description = "Preconfigured rules for SQLi, XSS, LFI, RCE, RFI, Scannerdetection, Protocolattack and Sessionfixation."
```

Comment:
Is this also for cloud armor? We should state that this is in combination with default_rules.